lib/actions.js

```diff
@@ -91,15 +93,18 @@ const webpackConfig = async ({
     output: {
       path: path.resolve(output || 'dist'),
       pathinfo: mode === 'development',
-      filename: 'deck.js' // TODO: Different name?
+      // TODO(5): Allow user-specified output bundle name?
+      // https://github.com/FormidableLabs/spectacle-cli/issues/5
+      filename: `deck${mode === 'production' ? '.min' : ''}.js`
```

Comment:
This is technically non-functional right now because production mode is disabled on all code paths because of #6